.gitignore

```
@@ -49,3 +49,4 @@ result
 /test/
 /iroha-java/
 /lcov.info
+**/test-smartcontracts/
```

Quite... indeterminate. I guess the target is always at the root?
```suggestion
/test-smartcontracts
```

---

it depends on from where you are running the script. if you are running from root then it will be in root yes

---

I believe it is better to leave it this way. maybe someone wanted to create a dir locally to test something so it makes sense if he ran the command within this dir to have test-smartcontracts inside that that dir as well.